modules/videoio/test/test_camera.cpp

```diff
@@ -104,5 +104,42 @@ TEST(DISABLED_videoio_camera, v4l_read_framesize)
 
     capture.release();
 }
+TEST(COUNT_OF_CAMERA_FRAMES, v4l_poll)
```

Comment:
`TEST(DISABLED_videoio_camera, v4l_poll)`

To run it call test application like this: 
```.sh
opencv_test_videoio --gtest_also_run_disabled_tests --gtest_filter=*v4poll
```